json/go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/urfave/cli-altsrc/json
```

Comment:
@dearchap should these be also on v3 or do you want to keep them loosely coupled?